src/index.js

```diff
@@ -6,7 +6,10 @@
 
 'use strict';
 
-var check = require('check-types');
+var check = require('check-types'),
+    project = require('./project'),
+    module = require('./module');
```

Comment:
This is my fault for calling a module "module", that was a really dumb idea on reflection. But anyway, for the sake of possible confusion down the line, even in a small file like this, I think we should probably avoid creating variable names that clash with global properties.